activiti-cloud-services-audit/activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/repository/EventsRepository.java

```diff
@@ -14,27 +14,30 @@
  * limitations under the License.
  */
 
-package org.activiti.cloud.services.audit;
+package org.activiti.cloud.services.audit.repository;
 
 import com.querydsl.core.types.dsl.StringPath;
+import org.activiti.cloud.services.audit.EventsRelProvider;
 import org.activiti.cloud.services.audit.events.QProcessEngineEventEntity;
 import org.activiti.cloud.services.audit.events.ProcessEngineEventEntity;
 import org.springframework.data.querydsl.QuerydslPredicateExecutor;
 import org.springframework.data.querydsl.binding.QuerydslBinderCustomizer;
 import org.springframework.data.querydsl.binding.QuerydslBindings;
 import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.rest.core.annotation.Description;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
-@RepositoryRestResource(exported = false)
+@RepositoryRestResource(path = EventsRelProvider.COLLECTION_RESOURCE_REL,
```

Comment:
@ffazzini I was about to merge it, but I've just realised that you forgot to override methods that are changing the database (we want the REST API to be read-only). See [Mongo implementation](https://github.com/Activiti/activiti-cloud-audit-service/blob/develop/activiti-cloud-services-audit/activiti-cloud-services-audit-mongo/src/main/java/org/activiti/cloud/services/audit/mongo/repository/EventsRepository.java#L16).